src/AmazonPHP/SellingPartner/ObjectSerializer.php

```diff
@@ -345,12 +351,12 @@ public static function deserialize(Configuration $configuration, $data, string $
 
         if (\method_exists($class, 'getAllowableEnumValues')) {
             $brokenModelDefinitions = [
-                \ltrim(EventCode::class, '/'), // https://github.com/amazon-php/sp-api-sdk/issues/191
-                \ltrim(ItemImage::class, '/'), // https://github.com/amazon-php/sp-api-sdk/issues/156
+                \ltrim(EventCode::class, '\\'), // https://github.com/amazon-php/sp-api-sdk/issues/191
+                \ltrim(ItemImage::class, '\\'), // https://github.com/amazon-php/sp-api-sdk/issues/156
```

Comment:
since there are already 2 places with those class names, could you please extract it to private method that would return this array?

Reply:
Yes, sir.
We will revise and push it tomorrow.